chain/bitcoind_zmq_events.go

```diff
@@ -358,9 +358,6 @@ func (b *bitcoindZMQEvents) txEventHandler() {
 				continue
 			}
 
-			// Add the tx to mempool.
```

Comment:
Typo in commit message: `LookupInputSpend` -> `LookupInputSpend`

Reply:
Also I don't follow how the second good case differs from the first bad case. What would cause the tx to be marked as seen in the bad case, but not the good case?

Reply:
I think this needs to be here otherwise the mempool will have confirmed txes stored?

Reply:
I think the next run of the mempool poller will get rid of it, which would make it not that much of an issue (still not ideal). Where does `filterTx` mark a tx as seen -- I can't find it?

Reply:
> Where does filterTx mark a tx as seen 

[Here](https://github.com/btcsuite/btcwallet/blob/master/chain/bitcoind_client.go#L1386) when it puts it into its(yet another...) mempool. And when this tx shows up again as unconfirmed it won't be notified because [here](https://github.com/btcsuite/btcwallet/blob/master/chain/bitcoind_client.go#L1301).